sklearn/ensemble/tests/test_weight_boosting.py

```diff
@@ -413,8 +471,7 @@ def fit(self, X, y, sample_weight=None):
         # Verify sparsity of data is maintained during training
         types = [i.data_type_ for i in sparse_classifier.estimators_]
 
-        assert all([(t == csc_matrix or t == csr_matrix)
-                   for t in types])
+        assert all([(t == csc_matrix or t == csr_matrix) for t in types])
```

Comment:
> Please try to avoid unrelated cosmestic changes like this (same below in `test_sparse_regression`).

You are right

>Also `test_sample_weight_adaboost_regressor` seems to have been moved a few lines up, is there any reason to that?

I think that is happened  because I removed and then readd the `test_sparse_classification` and `test_sparse_regression` tests

Reply:
It looks like now you should confirm that the format is unchanged, rather than always being converted to `csr_matrix`, yeah?

Reply:
A kind of that. What do you propose?

Reply:
I retain this test